client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -170,14 +170,14 @@ class PurchasesListing extends Component {
 			return null;
 		}
 
-		let label = translate( 'Manage Plan' );
+		let label = translate( 'Manage plan' );
 
 		if ( isJetpackBackup( purchase ) ) {
-			label = translate( 'Manage Product' );
+			label = translate( 'Manage product' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.